client/src/app.jsx

```diff
@@ -57,22 +61,22 @@ class App extends React.Component {
       });
   }
 
-  // Just for testing.  Can drop into a div using the following tag: onClick={this.logState} 
+  // Just for testing.  Can drop into a div using the following tag: onClick={this.logState}
```

Comment:
I think this is a good use of testing and comments, but ideally I think we would want to remove these functions that aren't necessary for correct functionality as well as extra comments before it is merged to master. In this case I believe it's okay to leave this in for now.

Reply:
You are right, especially about the merging into master part.  This is a bit of a edge case because I'm thinking I might need it for testing down the road.  Then again, if I do delete it, it's not that much work to create, and in the meantime it's added clutter.  I'll clean it up!